### DISCUSSION TITLE
How to bulk update the contents of an organisation?

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Knowlg/Community#213] [Q&A] [answered]
Hello team,

We've a scenario where we want to update the primary category and BMGS of 100+ contents of an particular organization.

Can we update them in bulk? How this can be achieved? 

cc: @komalm @manishakapadnis @paritshivani

**Comment by nidhikachhawa24** [comment]
Can we update directly in database?

Need urgent guidance.

**Comment by komalm** [comment]
Hello @pallakartheekreddy, @HarishGangula, @swayangjit @vinukumar-vs @rajeevsathish

We need your urgent guidance regarding the above critical point. We are waiting for your valuable insights to proceed further.

Thank you for your prompt attention to this matter.

cc: @nidhikachhawa24 @manishakapadnis @paritshivani

**Reply by pallakartheekreddy** [reply, marked as answer]
@komalm We don't have a bulk update API to handle this process. Direct updates to the database will not reflect in ECAR until the contents are re-published.
Please follow the steps below to achieve this:
1. Ensure none of the contents are in `Draft` status.
2. Use the search API to find the list of content do_ids (identifiers).
3. With the content do_ids, use the system update API to update the primary category and BMGS one by one. Consider writing a script (using Node.js) to automate this process.
4. After successfully updating the metadata, publish the contents using the publish API.

**Reply by nidhikachhawa24** [reply]
@pallakartheekreddy

Can we connect tomorrow morning to discuss briefly?

Please let me know the suitable time.

cc: @komalm @manishakapadnis @paritshivani

**Reply by nidhikachhawa24** [reply]
@pallakartheekreddy Can we connect today? 
We need urgent guidance as we are blocked on this.

cc: @komalm @manishakapadnis @paritshivani

**Reply by pallakartheekreddy** [reply]
@nidhikachhawa24 lets connect 6PM today.

**Reply by nidhikachhawa24** [reply]
@pallakartheekreddy 

Shared meeting invitation for 6PM

**Reply by pallakartheekreddy** [reply]
@nidhikachhawa24 I didn't receive any invitation. Please send it to kartheek@sanketika.in.

**Reply by pallakartheekreddy** [reply]
@nidhikachhawa24 @komalm Hope your query is now resolved. Please mark the thread as answered.
